package.json

```diff
@@ -5,8 +5,8 @@
   "scripts": {
     "lint": "jshint .",
     "test": "mocha",
-    "install": "node proxy/build-client.js",
-    "rebuild-client": "node proxy/build-client.js"
+    "rebuild-build": "node scripts/build.js",
+    "install": "npm run rebuild-build"
```

Comment:
Not sure if it's a good idea to run an `npm run` from inside an `npm run`.. it _should_ be fine, but I'm a little paranoid of the privilege dropping getting confused.